var/spack/repos/builtin/packages/py-cgal-pybind/package.py

```diff
@@ -9,16 +9,16 @@
 class PyCgalPybind(PythonPackage):
     """Internal Python bindings for CGAL"""
 
-    homepage = "https://bbpcode.epfl.ch/browse/code/common/cgal-pybind/tree/"
-    git      = "ssh://bbpcode.epfl.ch/common/cgal-pybind"
+    homepage = "https://bbpgitlab.epfl.ch/nse/cgal-pybind"
+    git = "git@bbpgitlab.epfl.ch:nse/cgal-pybind.git"
 
-    version('develop', submodules=True)
-    version('0.0.2', commit='7aa1382d1628ccd51f692750a2b145b1df0694d9', submodules=True)
+    version("develop", submodules=True)
+    version("0.1.0", tag="cgal_pybind-v0.1.0", submodules=True)
```

Comment:
I notice now that in `setup.py` the name does not follow the convention (it's came case instead of dash separated), and the tag is generated using it.
There is the same inconsistency in `poisson-recon-pybind`.
If it won't break anything because the packages are pretty new, it would be nice to fix them in the next version.
```
setup(
    name='cgal_pybind',
```

Reply:
Hi @GianlucaFicarelli, it's noted. I'll do these changes for the next versions of poisson-recon-pybind and cgal-pybind.

Reply:
@GianlucaFicarelli Will be done with https://bbpgitlab.epfl.ch/nse/cgal-pybind/-/merge_requests/6 and https://bbpgitlab.epfl.ch/nse/poisson-recon-pybind/-/merge_requests/8.

Reply:
Thanks!